src/sections/Footer/index.js

```diff
@@ -1,170 +1,219 @@
+import React from "react";
+import { Link } from "gatsby";
+import { TwitterTimelineEmbed } from 'react-twitter-embed' ;
+
+import { Container, Row, Col } from "../../reusecore/Layout";
+
+import { FiMail, FiPhoneCall } from "react-icons/fi";
+import { FaFacebookF, FaTwitter, FaLinkedinIn } from "react-icons/fa";
+
+import logo from "../../images/layer5/layer5.svg";
+import particleOne from "../../assets/images/app/footer/footer-particle.png";
+
+import mail_icon from "../..//assets/images/app/footer/social/mail_white_trim.svg";
+import slack_icon from "../../assets/images/app/footer/social/slack_white_trim.svg";
+import twitter_icon from "../../assets/images/app/footer/social/twitter_white_trim.svg";
+import github_icon from "../../assets/images/app/footer/social/github_white_trim.svg";
+import youtube_icon from "../../assets/images/app/footer/social/youtube_white_trim.svg";
+import docker_icon from "../../assets/images/app/footer/social/docker_white_trim.svg";
+
+
+import FotterWrapper from "./footer.style";
+import Language from "./languages";
+
+const Footer = () => {
+  return (
+    <FotterWrapper>
+      <img className="section__particle one" alt="img" src={particleOne} alt="appion app landing" />
+      <Container>
+        <Row>
+          <Col xs={12} sm={5} lg={4}>
+            <div className="first">
+              <Link className="footer-logo" to="#">
+                <img src={logo} alt="logo" />
+              </Link>
+              {/* <ul className="info">
+                <li>
+                  <FiMail />
+                  <Link className="anchor" to="#">
+                    contact@appion.com
+                  </Link>
+                </li>
+                <li>
+                  <FiPhoneCall />
+                  <Link className="anchor" to="#">
+                    +88 12345 697858
+                  </Link>
+                </li>
+              </ul> */}
+              <TwitterTimelineEmbed
+              sourceType="layer5"
+              screenName="Layer5"
+              options={{height: 300}}
+              noFooter
+              noScrollbar
+              />
+            </div>
+          </Col>
+          <Col xs={12} sm={7} lg={8}>
+          <Row>
+          <Col xs={12} sm={4} lg={5}>
+            <div className="footer-widgets">
+              <h3 className="widget-title">Resources</h3>
+              <ul className="widget-catagory">
+                <li>
+                  <Link className="anchor" to="#">
+                    Service Mesh Comparison
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Service Mesh Tools
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">
+                    Service Mesh Books
+                  </Link>
+                </li>
+                <li>
+                  <Link className="anchor" to="#">

```

Comment:
This needs to be collapsed with line 85 to be "Service Mesh Workshops". The link should be to `/workshops`

Reply:
The link can be done once we have a page there.

Reply:
Thanks for collapsing these into one. Looking good. 

Here's a pesky item that you might give some consideration to:

<img width="612" alt="Screen Shot 2020-08-02 at 11 21 42 PM" src="https://user-images.githubusercontent.com/7570704/89145966-e91a4b00-d517-11ea-8308-5a7e23819865.png">

Reply:
Updated all.